package.json

```diff
@@ -31,6 +32,7 @@
     "react-device-detect": "^2.2.3",
     "react-dom": "^18.2.0",
     "react-error-boundary": "^4.0.3",
+    "react-intl": "^6.4.7",
```

Comment:
maybe don't use `react-intl` if the app does not use it already

Reply:
You're right. Removed.